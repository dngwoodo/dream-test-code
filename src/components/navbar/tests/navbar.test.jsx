import React from 'react';

import renderer from 'react-test-renderer';

import Navbar from '../navbar';

describe('Navbar', () => {
  it('renders', () => {
    const component = renderer.create(<Navbar totalCount={3}/>);

    expect(component.toJSON()).toMatchSnapshot();
  });
});