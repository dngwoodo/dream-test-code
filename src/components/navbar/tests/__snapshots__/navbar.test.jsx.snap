// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders 1`] = `
<nav
  className="navbar"
>
  <i
    className="navbar-logo fas fa-leaf"
  />
  <span>
    Habit Tracker
  </span>
  <span
    className="navbar-count"
    data-testid="total-count"
  >
    3
  </span>
</nav>
`;
